src/pages/hegel/reference/nothing/development.mdx

```diff
@@ -52,16 +53,61 @@ Hegel goes to add a link between the empty thinking (or empty intuiting) that fo
+
+> utterly indeterminate _being_: for simple immediacy, at this stage in speculative logic, is all that being is understood to be (Houlgate 2022, 132).
+
+This sense of immediacy is further stressed through the self-equality of `nothing`. This simple quality with itself at once means that `nothing` is _immediately itself_. Houlgate additionally refers to §88 of Hegel's _Encyclopaedia Logic_, where, Hegel writes: "And similarly, but conversely, nothing, as this immediate [term] that is equal to itself, is the same as being" (Hegel 1991, 141/§88). And so Houlgate concludes that, "just as pure being, by virtue of its indeterminacy, vanishes into nothing, so pure nothing, by virtue of its immediacy, vanishes into being" (Houlgate 2022, 143).
+
+Houlgate additionally notes that while Hegel may seem to suggest that pure nothing turns to being because, as Hegel's text shows, "nothing _is_ (exists) in our intuiting or thinking", he reminds us that speculative logic does not provide an account of what happens to being and nothing when they are thought _by us_ human beings. The unfolding of these categories, their "fate" as it were, certainly requires human thought, but it is not human thought that determines what these categories are.
+
+> The development Hegel traces ... is not a product of our thinking; rather, it is made necessary, logically, by being and nothing themselves and is simply made explicit and articulated by thought. In speculative logic, our thought does not determinate the development that the categories of being and [nothing][^3] undergo, but it is itself determined by the immanent logical development of those categories (Houlgate 2022, 143).
+
+[^3]: The passage reads "being and thought" but thought is not a category, so this must be a textual error (Niklas).
+
+Houlgate thus states that it cannot be due to any operation of the mind, the understanding, intuition or some other cognitive faculty that determines the nature of the categories of being and nothing, but only these categories themselves in the instance they are thought purely through themselves.
+
+Incidentally, this immediacy of `nothing` serves to show that `nothing` is `being`, yet self-equality is a feature of `being` just as much as it is of `nothing`. How is it sufficient for `nothing` to be `being` in virtue of self-equality, when that is not the case for `being` to be `nothing`? Both categories are each self-equal and indeterminate, yet for only one of them does one of these features play a significant role: in the case of `nothing` it is, as we have seen, immediacy, and for, `being` it is indeterminacy. This is a question to further investigate when the [immediate difference between being and nothing]() is put to focus.
```

Comment:
Again, I'm not really sold on this reading. I shall put pen to paper!

Reply:
Verily, please do!